source/Octopus.Tentacle/Diagnostics/OctopusLogsDirectoryRenderer.cs

```diff
@@ -11,7 +11,7 @@ namespace Octopus.Tentacle.Diagnostics
     [LayoutRenderer("octopusLogsDirectory")]
     public class OctopusLogsDirectoryRenderer : LayoutRenderer
     {
-        public static readonly string DefaultLogsDirectory =
+        public static readonly string DefaultLogsDirectory = Environment.GetEnvironmentVariable("DefaultLogDirectory") ??
```

Comment:
Should this be placed in the EnvironmentVariables class as a const?

Reply:
Yep good idea 👍

Reply:
Done!